xiaogpt.py

```diff
@@ -34,6 +34,8 @@
 KEY_WORD = "帮我"
 PROMPT = "请用100字以内回答"
 
+# simulate the response from xiaoai server by type the input.
+CLI_INTERACTIVE_MODE=False
```

Comment:
add arg here?

Reply:
Should be , after another pr merged

Reply:
Or merge this first, and fix in another pr